example/lib/presentation/samples/bar/bar_chart_sample5.dart

```diff
@@ -188,6 +193,8 @@ class BarChartSample5State extends State<BarChartSample5> {
               value1 + value2,
               value1 + value2 + value3,
               AppColors.contentColorPink,
+                        '5',
```

Comment:
It seems that the formatting is not correct, as you can see we have many spaces on the left side of the `'5'`